apps/callscreen/js/call_screen.js

```diff
@@ -495,7 +500,7 @@ var CallScreen = {
     durationChildNode.textContent = '00:00';
     durationNode.classList.add('isTimer');
 
-    LazyL10n.get(function localized(_) {
+    navigator.mozL10n.ready(function() {
```

Comment:
`Utils.prettyDuration` is actually not using mozL10n.get, so you don't need ready here.